sql_generators/mobile_kpi_support_metrics/templates/active_users.view.sql

```diff
@@ -3,7 +3,10 @@ CREATE OR REPLACE VIEW
   `{{ project_id }}.{{ dataset }}.{{ name }}`
 AS
 SELECT
-  * EXCEPT (isp),
+  * EXCEPT (isp) REPLACE(
```

Comment:
why were we removing ISP from this view?

Reply:
we, include it at the bottom of the view definition. If I recall correctly and based on the comment there it's because in some of the products that column is in different position in comparison to others causing the downstream union across all products to fail.